frontend/sw360-portlet/src/main/webapp/html/projects/includes/linkedProjects.jspf

```diff
@@ -122,8 +72,29 @@
           <portlet:param name="<%=PortalConstants.PAGENAME%>" value="<%=PortalConstants.PAGENAME_SOURCE_CODE_BUNDLE%>"/> \
         </portlet:renderURL>';
     }
+
+    var linkedProjectsTable;
     $(window).load(function () {
-        $("#LinkedProjectsInfo").treetable({expandable: true});
+        linkedProjectsTable = $("#LinkedProjectsInfo");
+        linkedProjectsTable.treetable({
```

Comment:
That's right. I'm a little torn on whether it would be better to extract a common JS function from this. Logically, it'd ought to go into a new js file, but we make already too many requests to build one page as it is. Putting it into one of the existing js files would be misplacing, I think. Or do you know of a better place to put this code?
Also, there's #445 and maybe both trees will be unified some day.
Do you think I should extract the code nonetheless?